tox.ini

```diff
@@ -11,7 +11,7 @@ deps =
     django22: Django>=2.2,<2.3
     django30: Django>=3.0,<3.1
     drf39: djangorestframework<3.10
-    drflatest: djangorestframework
+    drflatest: djangorestframework<3.12
```

Comment:
Instead of marking this as the `drflatest`, can you create a `drf311` and use that in the `envlist` array instead?